Changed article "Get started with WebView2 in Windows Forms" (#1420)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/get-started/winforms.md

Start with a basic desktop project that contains a single main window.  

> 1. Open Visual Studio. In the opening panel, choose **Create a new project**.

Reviewer: ```suggestion
1. Open Visual Studio. In the opening panel, select **Create a new project**.
```
Source: public (or internal) version of MWSG, article "Describing interactions with UI", row: "Select" (& row "Choose"). 
 https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

    :::image type="complex" source="./media/winforms-opening-panel.png" alt-text="Visual Studio opening panel" lightbox="./media/winforms-opening-panel.png":::
       Visual Studio opening panel  
    :::image-end:::

> 1. Locate and to choose **C# Windows Forms .NET Framework App** then choose **Next**.

Reviewer: ```suggestion
1. Select **C# Windows Forms .NET Framework App**, and then select **Next**.
```

    
    :::image type="complex" source="./media/winforms-new-project.png" alt-text="New project" lightbox="./media/winforms-new-project.png":::
       New project  
    :::image-end:::
    

> 1.  Enter values for **Project name** and **Location**.  Choose **.NET Framework 4.7.2** or later.  

Reviewer: ```suggestion
1.  Enter values for **Project name** and **Location**.  Select **.NET Framework 4.7.2** or later.  
```

    
    :::image type="complex" source="./media/winforms-start-proj.png" alt-text="Start project" lightbox="./media/winforms-start-proj.png":::
       Start project  
    :::image-end:::
    

> 1.  Choose **Create**.

Reviewer: Global in this .md file: Change 'choose' to 'select'. 
Source: public (or internal) version of MWSG, article "Describing interactions with UI", row: "Select" (& row "Choose"). 
 https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui
Mgmt approved, in a previous conversation.

Add a WebView to your app.  

1. Choose **Project** > **Add Form (Windows Forms)...**.  
1. In the Add New Item panel, choose **Visual C# Items** > **Web** > **Windows Forms** > **Form (Windows Forms)** and choose **Add**.
1. Choose **View** > **Toolbox**.
1. In the **Toolbox** choose **WebView2 Windows Forms Control** to expand the options.  
    
    > [!NOTE]
    > If you are using Visual Studio 2017, by default **WebView2** may not display in the **Toolbox**. To enable the behavior, choose **Tools** > **Options** > **General** > and set the **Automatically Populate Toolbox** setting to `True`.  
    
1. Drag and drop the **WebView2** control into the Windows Forms App.
    
    :::image type="complex" source="./media/winforms-toolbox.png" alt-text="Toolbox displaying WebView2" lightbox="./media/winforms-toolbox.png":::
       Toolbox displaying WebView2  
    :::image-end:::  
    
1. Close the **Toolbox**.

> 1. In the Properties panel, set the **(Name)** property to **webView**. Use the **Categorized** and **Alphabetical** sort options as needed to find properties.

Reviewer: ```suggestion
1. In the **Properties** panel, set the **(Name)** property to **webView**. Use the **Categorized** and **Alphabetical** sort options as needed to find properties.
```

    
    :::image type="complex" source="./media/winforms-properties.png" alt-text="Properties of the WebView2 control" lightbox="./media/winforms-properties.png":::
       Properties of the WebView2 control
    :::image-end:::
    
1.  The **Source** property sets the initial URI displayed in the WebView2 control. Set the **Source** property to `https://www.microsoft.com`.

> 1. Select **F5** to build and run your project.

Reviewer: One philosophy of doc'ing UI is to give GUI instructions (verbose version: "Select the Foo menu and then select Blah.  or:  "Select Foo > Bar.")  The UI shows the shortcut keys so the docs don't need to mention shortcut keys.  Examples in MWSG:  https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

    Make sure the WebView2 control displays [https://www.microsoft.com][MicrosoftMain].

    :::image type="complex" source="./media/winforms-hello-webview.png" alt-text="hello webview" lightbox="./media/winforms-hello-webview.png":::
       hello webview  
    :::image-end:::

    > [!NOTE]
    > If you are working on a high DPI monitor, you may have to [configure your Windows Forms app for high DPI support][DotnetFrameworkWinformsHighDpiSupportWindowsFormsConfiguringYourWindowsFormsAppForHighDpiSupport].  
    
## Step 4 - Add controls and process window resize events  

> Add more controls to your Windows Forms from the toolbox, then process window resize events.  

Reviewer: ```suggestion
Add more controls to your form from the toolbox, then process window resize events.  
```

1. Choose **View** > **Toolbox**.
1. In the **Toolbox** choose **Common Controls**. 
1. Drag and drop **TextBox** control into the Windows Forms App.
1. In the Properties panel, change the **\(Name\)** to **addressBar**.  

> 1. Drag and Drop a **Button** control into the Windows Forms App.

Reviewer: "Drag and drop", or just "Drag foo into bar."  Check https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

> 1. In the Properties panel, change the **\(Name\)** to **goButton**.

Reviewer: ```suggestion
1. In the **Properties** panel, change the **\(Name\)** to **goButton**.
```

1. Change the **Text** property to **Go!**.
1. Resize the button as needed to display the text.
1. Arrange the text box to the left of the button, aligned by the text as shown. 
    
    :::image type="complex" source="./media/winforms-designer.png" alt-text="WinForms designer" lightbox="./media/winforms-designer.png":::
       WinForms designer  
    :::image-end:::  

1. Resize the text box as shown.

    :::image type="complex" source="./media/winforms-designer-txtbtn.png" alt-text="WinForms designer textbox and button" lightbox="./media/winforms-designer-txtbtn.png":::
       WinForms designer textbox and button
    :::image-end::: 

1. Choose **View** > **Code** to open the `Form1.cs` file.

>     Define `Form_Resize` to keep the controls in place when the App Window is resized.

Reviewer: lowercase as "app window"?  If title bar shows **App Window** label, maybe use bold with casing like the UI.

To continue learning more about WebView2, navigate to the following resources.  

> *   Building WebView2 applications: [WebView2 development best practices][WV2BestPractices].  
> *   Comprehensive example of WebView2 capabilities: [WebView2Samples][GithubMicrosoftedgeWebview2samplesMain].  
> *   More information about WebView2: [WebView2 Resources][Webview2IndexNextSteps].  
> *   Detailed information about the WebView2 API: [API reference][DotnetApiMicrosoftWebWebview2WinformsWebview2].  

Reviewer: ```suggestion
*  [WebView2 development best practices][WV2BestPractices] - Building WebView2 applications.
*  [WebView2Samples][GithubMicrosoftedgeWebview2samplesMain] - A comprehensive example of WebView2 capabilities.
*   [WebView2 Resources][Webview2IndexNextSteps] - More information about WebView2.
*   [API reference][DotnetApiMicrosoftWebWebview2WinformsWebview2] - Detailed information about the WebView2 API.
```
A suggested alternative layout.
